frontend/src/Editor/Box.jsx

```diff
@@ -284,13 +284,13 @@ export const Box = function Box({
         style={{
           ...styles,
           backgroundColor,
-          boxShadow: validatedGeneralStyles?.boxShadow,
         }}
         role={preview ? 'BoxPreview' : 'Box'}
       >
         {inCanvas ? (
           !resetComponent ? (
             <ComponentToRender
+              boxShadow={validatedGeneralStyles?.boxShadow}
```

Comment:
@stepinfwd just curious can we send boxShadow as a part of styles props itself since styles is already being sent to every component as prop. This can save us from passing another prop (boxShadow).

<ComponentToRender
 styles = {{...validatedStyles, boxShadow : validatedGeneralStyles?.boxShadow}}
 />